docs/deprecated.md

```diff
@@ -122,6 +123,14 @@ The `AUTOMATED` column has been removed from the default `docker search`
 and `docker image search` output in v25.0, and the corresponding `IsAutomated`
 templating option will be removed in v26.0.
 
+### Logentries logging driver
+
+**Target For Removal In Release: v25.0**
```

Comment:
Add deprecated in

Reply:
Ah, yes, I initially left it out, because I wasn't sure what version to pick; the service was discontinued and we never called it out explicitly to be deprecated because of that, so I picked "v24.0" for the table, but the most truthful would be "November 15, 2022" when the service was discontinued.

Let me add v24.0